qa/automation/CareerPortal.Tests.Scenarios/Savostin/buttons.feature

```diff
@@ -0,0 +1,52 @@
+Feature: 1.0.0 buttons lead to correct pages
+Background: 
+	Given "Career Quantori" page is opened
+
+Scenario Outline: 1.0.0_01 All buttons of header lead to correct pages
+	When user clicks on <button>
+	Then <page> is opened
+
+	Examples:
+	|button            |page                                 |
+	|About             |https://career.quantori.com/about    |
+	|Open job positions|https://career.quantori.com/positions|
+	|Contact           |https://career.quantori.com/contact  |
+
+Scenario Outline: 1.0.0_02 All buttons of footer lead to correct pages
+	When user clicks on <button>
+	# VPN must be active
+	Then <page> is opened
+
+	Examples:
+	|button         |page                                                                             |
```

Comment:
I would recommend to add 2 more buttons: Quantori logo in the bottom left corner and "Data Privacy Policy" as these two are also included in footer

Reply:
Done! You can find it in buttons_of_Career_Quantori_page.feature 1.0.0_02